element-count.js

```diff
@@ -0,0 +1,16 @@
+/* eslint no-console: "off" */
+/* exported tryQuerySelector */
+'use strict';
+
+function tryQuerySelector() {
+    var selectedElements = document.querySelectorAll('input[name="elements"]:checked');
+    var results = document.getElementById('results');
+
```

Comment:
```suggestion
    results.textContent = '';
```
By adding this, every time tryQuerySelector is called, it will reset the text content to an empty string. Without this, every time the user clicks guess, the text content just gets longer and longer.